src/core/positioning/positioning.h

```diff
@@ -305,6 +331,12 @@ class Positioning : public QObject
     bool mBackgroundMode = false;
 
     QVariantMap mPropertiesToSync;
+
+    bool mAveragedPosition = false;
+    QList<GnssPositionInformation> mCollectedPositionInformations;
```

Comment:
Are these members (and getter/setter) still in positioningsource.cpp/h? If so they need to be taken away.